frontend/talentsearch/package.json

```diff
@@ -25,10 +25,9 @@
     "lint:fix": "eslint ./src/js --fix --ext .js,.ts,.jsx,.tsx",
     "prettier": "prettier --write ./src/js",
     "dev": "npx webpack",
-    "watch": "mix watch",
-    "watch-poll": "mix watch -- --watch-options-poll=1000",
-    "hot": "mix watch --hot",
-    "production": "mix --production",
+    "watch": "npx webpack watch",
+    "hot": "npx webpack watch --hot",
+    "production": "npx webpack --production",
```

Comment:
consider removing npx usage from all npm run-script places in this repo

Reply:
Good catch!